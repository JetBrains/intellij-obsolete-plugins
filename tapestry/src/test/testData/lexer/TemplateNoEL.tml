<html xmlns:t="http://tapestry.apache.org/schema/tapestry_5_1_0.xsd">
    <head>
    </head>
    <body>
        <a href="#" t:type="pagelink" page="startpage">my link</a>
    </body>
</html>
