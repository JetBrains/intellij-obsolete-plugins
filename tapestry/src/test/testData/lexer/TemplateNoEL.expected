XML_START_TAG_START ('<')
XML_NAME ('html')
WHITE_SPACE (' ')
XML_NAME ('xmlns:t')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('http://tapestry.apache.org/schema/tapestry_5_1_0.xsd')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_START_TAG_START ('<')
XML_NAME ('head')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_END_TAG_START ('</')
XML_NAME ('head')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_START_TAG_START ('<')
XML_NAME ('body')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
        ')
XML_START_TAG_START ('<')
XML_NAME ('a')
WHITE_SPACE (' ')
XML_NAME ('href')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('#')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
WHITE_SPACE (' ')
XML_NAME ('t:type')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('pagelink')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
WHITE_SPACE (' ')
XML_NAME ('page')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('startpage')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_DATA_CHARACTERS ('my')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('link')
XML_END_TAG_START ('</')
XML_NAME ('a')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_END_TAG_START ('</')
XML_NAME ('body')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
')
XML_END_TAG_START ('</')
XML_NAME ('html')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
')
